tests/e2e/vsphere_file_volume_basic.go

```diff
@@ -485,18 +549,22 @@ func testHelperForCreateFileVolumeFailWhenFileServiceIsDisabled(f *framework.Fra
 	}()
 
 	ginkgo.By("Expect claim fails to provision volume when file service is disabled")
-	err = fpv.WaitForPersistentVolumeClaimPhase(v1.ClaimBound, client, pvclaim.Namespace, pvclaim.Name, framework.Poll, time.Minute/2)
+	err = fpv.WaitForPersistentVolumeClaimPhase(v1.ClaimBound, client,
+		pvclaim.Namespace, pvclaim.Name, framework.Poll, time.Minute/2)
 	gomega.Expect(err).To(gomega.HaveOccurred())
 	expectedErrMsg := "failed to provision volume with StorageClass \"" + storageclass.Name + "\""
 	ginkgo.By(fmt.Sprintf("Expected failure message: %+q", expectedErrMsg))
-	isFailureFound := checkEventsforError(client, namespace, metav1.ListOptions{FieldSelector: fmt.Sprintf("involvedObject.name=%s", pvclaim.Name)}, expectedErrMsg)
+	isFailureFound := checkEventsforError(client, namespace,
+		metav1.ListOptions{FieldSelector: fmt.Sprintf("involvedObject.name=%s", pvclaim.Name)}, expectedErrMsg)
 	gomega.Expect(isFailureFound).To(gomega.BeTrue(), "Unable to verify pvc create failure")
 }
 
-func createFileVolumeUsingDatastoreFromVsphereConf(f *framework.Framework, client clientset.Interface, namespace string, accessMode v1.PersistentVolumeAccessMode) {
+func createFileVolumeUsingDatastoreFromVsphereConf(f *framework.Framework,
+	client clientset.Interface, namespace string, accessMode v1.PersistentVolumeAccessMode) {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
-	ginkgo.By("Invoking test to check if the TargetvSANFileShareDatastoreURLs in vSphere config is applied when no datastore is specified in Storage Class")
+	ginkgo.By("Invoking test to check if the TargetvSANFileShareDatastoreURLs in vSphere config is applied " +
```

Comment:
Did you run "go fmt"?  I believe there should be no space before +

Reply:
Yes, I always run "make check".

Golang is weird. If the string is the only argument, spaces are needed around "+". Otherwise, not.

Reply:
Hmm, I'm not getting this:(.

Reply:
Let me use examples.
1) func("string 1" + "string 2") // Spaces are required around "+" in this case.
2) func("string 1"+"string 2", parameter2) // Spaces are not needed around "+" in this case.